scripts/kubestellar

```diff
@@ -256,31 +259,42 @@ function ensure_espw() {
     echo "Finished populating the espw with kubestellar apiexports"   
 }
 
-if [ "$subcommand" == init ]; then
-    ensure_root_compute_configd
-    ensure_espw
-    exit
-fi
-
-# Check mailbox-controller is already running
-if [ $(process_running mailbox-controller) == "running" ]
-then
-    echo "An older deployment of mailbox-controller is already running - terminating it ...."
-    pkill -f mailbox-controller
-fi
-
-# Check kubestellar-where-resolver is already running
-if [ $(process_running "kubestellar-where-resolver") == "running" ]
-then
-    echo "An older deployment of kubestellar-where-resolver is already running - terminating it ...."
-    pkill -f "kubestellar-where-resolver"
-fi
+# current ws at start does not matter, is imw1 on return
+function ensure_imw1() {
+    kubectl ws root &> /dev/null
+    if kubectl get ws imw1 &> /dev/null
+    then kubectl ws imw1 &> /dev/null
+    else kubectl ws create imw1 --enter &> /dev/null
+    fi
+    
+    if ! kubectl get apibinding "edge.kubestellar.io" &> /dev/null; then
+	kubectl kcp bind apiexport root:espw:edge.kubestellar.io
+    fi
+}
 
-# Check placement-translator is already running
-if [ $(process_running placement-translator) == "running" ]
-then
-    echo "An older deployment of placement-translator is already running - terminating it ...."
-    pkill -f placement-translator
+if [ "$subcommand" != init ]; then
+    
+    # Check mailbox-controller is already running
+    if [ $(process_running mailbox-controller) == "running" ]
+    then
+	echo "An older deployment of mailbox-controller is already running - terminating it ...."
+	pkill -f mailbox-controller
+    fi
+    
+    # Check kubestellar-where-resolver is already running
+    if [ $(process_running "kubestellar-where-resolver") == "running" ]
+    then
+	echo "An older deployment of kubestellar-where-resolver is already running - terminating it ...."
+	pkill -f "kubestellar-where-resolver"
+    fi
+    
+    # Check placement-translator is already running
+    if [ $(process_running placement-translator) == "running" ]
+    then
+	echo "An older deployment of placement-translator is already running - terminating it ...."
+	pkill -f placement-translator
+    fi
+    
 fi
 
 if [ $subcommand == stop ]; then    
```

Comment:
Not related to this PR. I find it confusing to have an explicit "stop" subcommand that is based on the fact that you kill the process by default ..

Reply:
That was a design choice made long ago and not part of this PR. Feel free to open an issue or PR if you want to suggest a better implementation or a change to the design for the set of platform management commands.